api/app/settings/local.py

```diff
@@ -2,6 +2,9 @@
 
 from app.settings.common import *
 
+os.environ["ENABLE_AXES"] = "False"
```

Comment:
Yuck! This is the same as just doing `ENABLE_AXES = False`, let's not modify environment variables in the code.

Reply:
Fixed